### ISSUE TITLE
categories and frontend

### ISSUE THREAD

**Issue by andreaskastl** [lochmueller/calendarize#469]
Hi, 

I miss the functionality to filter the frontend plugin output by categories, i.e. show only events from specified category/categories on this page. 

When I add the calendarize plugin to a page, there is no category field within plugin configuraton where I can specify the relevant categories.

It seems this functionality was already there in a previous version, but does not work as expected with current version (Typo3 10.4.10, Calendarize 7.3).

See also 
https://github.com/lochmueller/calendarize/issues/44
https://github.com/lochmueller/calendarize/issues/9

Please provide this functionality (again). It would help a lot to migrate from cal to calendarize in a multi site environment.

Regards
Andreas

**Comment by lochmueller** [comment]
Hey @andreaskastl ,
do you try the regular categories of the content Element?
Every Content Element has categories. I think calendarize use this categories....
Regards,
Tim

**Comment by andreaskastl** [comment]
Hi @lochmueller,

I also tried to use these categories of the content element itself, but without any result.

I have 4 test entries
- Event1 with Category1 and Category2 assigned
- Event2 with Category1
- Event3 with Category2
- Event4 without any Category

On a test page I added the Calendarize frontend plugin with list view mode and selected in the category tab of the content element "Category1". Opening the page in frontend results in "all events are shown" => no category filter was considered.
Switching the category to Category2 has same effect => all events are shown.

Regards
Andreas
